packages/rn-tester-e2e/package.json

```diff
@@ -11,15 +11,24 @@
   "directory": "packages/rn-tester-e2e"
   },
   "scripts": {
-    "test-e2e": "node ./../../scripts/e2e/run-e2e-tests.js"
+    "test-e2e-ios": "npx wdio wdio.conf.ios.js",
+    "test-e2e-android": "npx wdio wdio.conf.android.js"
   },
   "devDependencies": {
-    "eslint": "^8.19.0",
-    "jest": "^29.6.3",
+    "eslint": "^8.23.1",
     "webdriverio": "^7.32.0",
     "@babel/core": "^7.20.0",
     "@babel/preset-env": "^7.20.0",
     "@babel/plugin-transform-flow-strip-types": "^7.20.0",
-    "@types/jest": "^29.5.3"
+    "@babel/register": "^7.20.0",
+    "@types/mocha": "^10.0.3",
+    "@wdio/appium-service": "^7.32.0",
+    "@wdio/cli": "^7.32.0",
+    "@wdio/local-runner": "^7.32.0",
+    "@wdio/mocha-framework": "^7.32.0",
+    "@wdio/spec-reporter": "^7.32.0",
+    "appium": "2.0.0",
```

Comment:
We're also installing `appium` globally, IIRC we should do it only in one place.